src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -100,7 +100,7 @@ HttpStatusCode mongoSubscribeContext
     bool notificationDone = false;
     BSONArray conds = processConditionVector(&requestP->notifyConditionVector,
                                              requestP->entityIdVector,
-                                             requestP->attributeList, oid.str(),
+                                             requestP->attributeList, oid.toString(),
```

Comment:
``` C++
class OID {
...
        std::string str() const { return toHexLower(data, kOIDSize); }
        std::string toString() const { return str(); }
....
```
It's just an alias, but in some cases is more portable because does not matches with str operator.
Some months ago I've got trouble to compile that 4.8.2, but now it work with 4.8.3 :S